spec/models/candidate_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe Candidate, type: :model do
+    it { should belong_to(:poll) }
+    it { should validate_presence_of(:title) }
+    it { should have_many(:votes).dependent(:destroy) }
+
+    describe "#calculate_result" do
+        let(:poll) { Poll.create!(title: "Poll", start_time: "2022-09-29T16:31", end_time: "2022-09-29T14:34") }
```

Comment:
Your spacing seems to be off in all your ruby files, ruby should always be 2 spaces, yours seems like its 4?